{"cve": {"id": "CVE-2024-42992", "sourceIdentifier": "cve@mitre.org", "published": "2024-08-23T18:15:07.427", "lastModified": "2024-08-26T03:35:00.920", "vulnStatus": "Awaiting Analysis", "cveTags": [], "descriptions": [{"lang": "en", "value": "Python Pip Pandas v2.2.2 was discovered to contain an arbitrary file read vulnerability."}], "metrics": {"cvssMetricV31": [{"source": "134c704f-9b21-4f2e-91b3-4a467353bcc0", "type": "Secondary", "cvssData": {"version": "3.1", "vectorString": "CVSS:3.1/AV:L/AC:L/PR:L/UI:R/S:U/C:N/I:N/A:N", "attackVector": "LOCAL", "attackComplexity": "LOW", "privilegesRequired": "LOW", "userInteraction": "REQUIRED", "scope": "UNCHANGED", "confidentialityImpact": "NONE", "integrityImpact": "NONE", "availabilityImpact": "NONE", "baseScore": 0.0, "baseSeverity": "NONE"}, "exploitabilityScore": 1.3, "impactScore": 0.0}]}, "references": [{"url": "https://github.com/juwenyi/CVE-2024-42992", "source": "cve@mitre.org"}, {"url": "https://pandas.pydata.org/", "source": "cve@mitre.org"}]}}